doc/manual/interp.rst

```diff
@@ -0,0 +1,63 @@
+.. _interp:
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+
+Repairing bad channels
+######################
```

Comment:
here you go: https://dl.dropboxusercontent.com/u/3915954/html/manual/interp.html